app/routes/press-releases/overview/planned.js

```diff
@@ -1,4 +1,21 @@
 import Route from '@ember/routing/route';
+// eslint-disable-next-line ember/no-mixins
+import DataTableRouteMixin from 'ember-data-table/mixins/route';
 
-export default class PressReleasesOverviewPlannedRoute extends Route {
-}
+export default class PressReleasesOverviewPlannedRoute extends Route.extend(DataTableRouteMixin) {
+  modelName = 'press-release';
+
+  mergeQueryOptions(params) {
+    const queryParams = {
+      sort: params.sort,
+      include: 'publication-event',
+      filter: {
+        ':has:publication-event': 'yes',
+      },
+    };
+
+    queryParams['filter[publication-event][:has:planned-start-date]'] = 'yes'
```

Comment:
De `:has:` filter kan enkel gebruikt worden voor relaties, niet voor attributes. In [de documentatie van de API](https://github.com/mu-semtech/mu-cl-resources#special-filters) staan alle speciale filters toegelicht. Om te kijken of een datum in de toekomst ligt, moet je gebruik maken van `:gt:` (greater than) en vergelijken met de huidige datum/tijd.

Is er een specifieke reden om deze filter op een aparte lijn te definiëren en niet mee op te nemen in het `queryParams` object hierboven?

Reply:
spijtig genoeg lukt het mij nog steeds niet.
dit is de foutmelding dat ik krijg bij beide filters (lt en gt)
![image](https://user-images.githubusercontent.com/54099103/128276432-ff2232ac-98eb-400a-bf13-645d7fad2865.png)

lijn 17 geen speciefieke reden, puur voor testen, omdat dit sowieso nog aangepast moest worden.